examples/args_test.rn

```diff
@@ -4,5 +4,5 @@ parser = Argparser.Argparser()
 parser.add_flag("--version", "-v", "Show version")
 parser.add_named("--output", "Output file")
 parser.add_pos_opt("filename", "File")
-print(parser.parse(["./program.rn", "a", "--output", "b"]))
+print(parser.parse(argv))
```

Comment:
Where `argv` is coming from?

Reply:
[https://github.com/angelcaru/radon/blob/182b7ceb5f81d1b0caff26f0d215924c5da15391/radon.py#L101](radon.py:101)